packages/backend/src/graphql/schema.graphql

```diff
@@ -183,8 +183,10 @@ type ActionSubstepArgument {
   variables: Boolean
   variableTypes: [String]
   allowArbitrary: Boolean
+  isSearchable: Boolean
```

Comment:
can i check is this just for the pointer icon to be displayed if it is false? for sanity purposes, should we also include this in the `TriggerSubstepArgument` in case we forget

Reply:
I feel like we just add in if necessary